### PR TITLE
sketch in v1 of site using next.js

### PR DESCRIPTION
**Pull request by tkellen**
Images will be removed and pulled from CDN before this lands.

### CODE REVIEW

File: projects/woodlandkitchen/v1/components/hamburger-button.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
+/*!
```
Comment (tkellen) [inline projects/woodlandkitchen/v1/components/hamburger-button.js]: @gnarf, this is the component I was talking with you about earlier today. Curious to hear what you think if you have time to take a peek.

File: projects/woodlandkitchen/v1/components/nav.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import { Component } from 'react'
```
Comment (tkellen) [inline projects/woodlandkitchen/v1/components/nav.js]: @gnarf, here is where the aforementioned button is consumed.

File: projects/woodlandkitchen/v1/components/nav.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+            }
+            a {
+              padding: 1vw;
+              border: none;
+            }
+          }
+        `}</style>
+        <HamburgerButton clickHandler={this.showHide}/>
+        <ul className={forceShowClass}>
+          <li><Link href="/about"><a>about</a></Link></li>
```
Comment (tkellen) [inline projects/woodlandkitchen/v1/components/nav.js]: ...going to pull these out of the component eventually.

**Comment by gnarf** [comment]
This is a pretty interesting way of doing CSS.

**Comment by tkellen** [comment]
cc @mzgoddard, you might find this interesting

File: projects/woodlandkitchen/v1/components/Panel.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+      a:hover {
+        color: #336627;
+      }
+      p {
+        position: absolute;
+        width: 100%;
+        bottom: 0;
+        text-align: center;
+        background-color: rgba(0,0,0,0.8);
+        border-top: 2px solid #000;
```
Comment (tkellen) [inline projects/woodlandkitchen/v1/components/Panel.js]: use theme color.

**Comment by tkellen** [comment]
@gnarf I figured out another approach to getting the display to work the way I wanted. I had to shift my thinking to a world where I CANT change the style of children. I am very curious to see how many different ways this limitations changes the way I implement the site, and if it is actually possible to do in a clean way.

If you're curious about the details hit me up and I'll talk 'em through with you. I don't want to take the time to try to write down what I did.

**Pull request merged**